src/tasks/task.dto.ts

```diff
@@ -0,0 +1,5 @@
+export interface TaskDto {
+  title: string;
+  description: string;
```

Comment:
I think in Prisma schema you have description marked as nullable, why you are not following the typing here?

Reply:
description?: string;